### ISSUE TITLE
Expose to Advanced Search on Semantic Nodes

### ISSUE THREAD

**Issue by azaroth42** [archesproject/arches#6176]
**Is your feature request related to a problem? Please describe.**

In trying to optimize both loading and searching, I am turning off a lot of fields that will never be searched for. This is super super time consuming for large models full of boilerplate, hidden fields.

**Describe the solution you'd like**
Allow the Expose to Advanced Search toggle on semantic nodes, and use as a shortcut to turn off or on all nodes below that semantic node.

**Comment by robgaston** [comment]
This will require some thought - it assumes that all cards are rooted in a semantic node which is not necessarily the case.

Perhaps there is a better (more explicit) way to handle this at the card level...?

**Comment by azaroth42** [comment]
True.  A separate toggle that turns on/off per branch, regardless of data-type?

**Comment by robgaston** [comment]
I'm thinking something like that: a toggle at the card level to hide the card and all it's children from advanced search
